changelog/v1.6.0-beta10/invalid-vs-gateway.yaml

```
@@ -0,0 +1,6 @@
+changelog:
+  - type: FIX
+    issueLink: https://github.com/solo-io/gloo/issues/3538
+    description: >- 
+      Fixed a bug where a bad authconfig which should invalidate
+      a single vs was incorrectly invalidating the entire gateway
```

let's use the long forms of the names in the changelog,
```suggestion
      a single virtual service was incorrectly invalidating the entire gateway 
```